crates/graphql/src/schema/query_root.rs

```diff
@@ -101,11 +111,31 @@ impl QueryRoot {
         wallet: PublicKey<Wallet>,
         limit: i32,
         offset: i32,
+        exclude_types: Option<Vec<String>>,
     ) -> FieldResult<Vec<FeedEvent>> {
         let conn = ctx.shared.db.get().context("failed to connect to db")?;
 
-        let feed_events =
-            queries::feed_event::list(&conn, wallet, limit.try_into()?, offset.try_into()?)?;
+        let exclusion_types_parsed: Option<Vec<EventType>>;
```

Comment:
At some point this should probably work similarly to `PublicKey`, such it is automagically converted from a Graph QL enum to a rust enum, so that the query UI can show valid values.